CONTRIBUTING.md

```diff
@@ -145,3 +145,49 @@ to their customers), *end user* (who use and consume KServe) and *integrations*
 that work with KServe).
 
 Please add your company logo, preferably in SVG format, to [here](https://github.com/kserve/website/tree/main/docs/images).
+
+## Becoming a committer
+
+A committer is a contributor who has gained the privilege to commit code to the project. 
+But being a committer also means being committed to the project and the community as a whole.
+
+**Responsibilities of a committer:**
+
+- Triage and respond to issues
+- Be an expert in certain areas, but familiar with the majority of the code base
```

Comment:
Let's discuss and define the areas, currently I am thinking of
- KServe control plane
- KServe open inference protocol, data plane
- KServe python SDK
- KServe release & testing infra
- ModelMesh
- KServe webapp
- KServe website

Reply:
I added another section to outline the ficus areas, areas of expertise